src/java.desktop/share/classes/javax/swing/plaf/synth/SynthLookAndFeel.java

```diff
@@ -1041,4 +1041,34 @@ private void repaintIfBackgroundsDiffer(JComponent comp) {
             }
         }
     }
+
+    // Toggle flag for drawing the mnemonic state
+    private static boolean isMnemonicHidden = true;
```

Comment:
I wonder why it defaults to `true` where only one L&F derived from Synth sets it to true.

Reply:
That's correct. Modified it to be `false` by default and based on L&F it can be set to `true`. For GTK initialization, it is set to `true` to hide mnemonics by default.